qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -113,6 +113,9 @@ def main(self):
         self.add_options(parser)
         (self.options, self.args) = parser.parse_args()
 
+        if os.getenv("QT", ""):
+            self.options.noshutdown = True
```

Comment:
> I think the noshutdown flag is appropriate.

Still, you could just pass it via the command line. On the other hand, there is no way to disable it, right now when using `QT=`.